src/house.cpp

```diff
@@ -172,10 +172,10 @@ void House::setAccessList(uint32_t listId, std::string_view textlist)
 	}
 
 	// kick uninvited players
-	for (HouseTile* tile : houseTiles) {
+	for (const auto& tile : houseTiles) {
 		if (CreatureVector* creatures = tile->getCreatures()) {
 			for (int32_t i = creatures->size(); --i >= 0;) {
-				Player* player = (*creatures)[i]->getPlayer();
+				auto player = (*creatures)[i]->getPlayer();
```

Comment:
Could this be `const auto`
```suggestion
				cont auto player = (*creatures)[i]->getPlayer();
```